home assignment/6/task1.py

```diff
@@ -0,0 +1,10 @@
+# 1 – Напишите программу вычисления арифметического выражения заданного строкой. Используйте операции +,-,/,*. приоритет операций стандартный.
+# Тупой вариант решения задачи:
+
+from sympy import sympify
+
+string = str(input('Введите ваше арифметическое выражение: '))
```

Comment:
```suggestion
string = input('Введите ваше арифметическое выражение: ')
```
Функция ввода данных и так возвращает строку.